front/src/components/catalog/Catalog.tsx

```diff
@@ -1,26 +1,18 @@
-import { NFTCard } from "../../schema/NFTCatalogSchema";
-import React from "react";
-import NFTMicroCard from "../nft-micro-card/NFTMicroCard";
+import React, { ReactNode } from "react";
 import { useStyles } from "./CatalogStyles";
-import Link from "next/link";
-import { FRONT_CURRENT_PATH } from "../../config/env-paths";
 
 interface CatalogProps {
-  nfts: NFTCard[];
+  children: ReactNode;
+  layout: string;
+  className?: string;
 }
 
-const Catalog = ({ nfts }: CatalogProps) => {
+const Catalog = ({ children, layout, className }: CatalogProps) => {
   const { classes } = useStyles();
 
   return (
-    <div className={classes.catalog}>
-      {nfts.map((item: NFTCard) => (
-        <Link href={FRONT_CURRENT_PATH + ":3000/nft/" + item.nftCardId}>
-          <div className={classes.card} key={item.nftCardId}>
-            <NFTMicroCard item={item} />
-          </div>
-        </Link>
-      ))}
+    <div className={className}>
+      <div className={classes.catalogOfFour}>{children}</div>
```

Comment:
thats not gonna work after merge. you should use layout as a parameter here - please, take a look at big_daddy_branch as its usage